src/Commands/RunRepeater.ts

```diff
@@ -96,13 +111,27 @@ export class RunRepeater implements CommandModule {
       .conflicts('remove-daemon', 'daemon')
       .env('REPEATER')
       .exitProcess(false)
+      .middleware(async (args: Arguments) => {
+        if (args.cert) {
+          try {
+            await Promise.all(
+              Object.values(args.cert as Record<string, string>).map((cert) =>
+                fileExists(cert)
+              )
+            );
+          } catch (e) {
+            throw new Error(`Certificate ${e.path} not found.`);
+          }
+        }
+      })
```

Comment:
It doesn't solve the issue. You have to check certs on each request. See https://github.com/NeuraLegion/nexploit-cli/pull/182#discussion_r602115113